Formula/llvm@7.rb

```diff
@@ -17,7 +17,8 @@ class LlvmAT7 < Formula
 
   keg_only :versioned_formula
 
-  deprecate! date: "2022-05-29", because: :versioned_formula
+  # Deprecated in homebrew since 2022-05-29
+  disable! date: "2022-06-30", because: :versioned_formula
```

Comment:
We'll have to wait a bit, I think.

> Formulae should not be disabled without a deprecation period of at least three months unless the circumstances are exceptional (e.g. the formula does not build on any supported macOS version or Linux).

https://docs.brew.sh/Deprecating-Disabling-and-Removing-Formulae#disabling

Reply:
Let's do this then.
```suggestion
  disable! date: "2022-09-30", because: :versioned_formula
```